packages/stryker/src/child-proxy/ChildProcessProxy.ts

```diff
@@ -51,13 +51,13 @@ export default class ChildProcessProxy<T> {
   }
 
   private initProxy() {
-    Object.keys(this.constructorFunction.prototype).forEach((methodName: keyof T) => {
-      this.proxyMethod(methodName);
+    Object.keys(this.constructorFunction.prototype).forEach(methodName => {
+      this.proxyMethod(methodName as keyof T);
     });
   }
 
-  private proxyMethod(methodName: keyof T) {
-    this.proxy[methodName] = (...args: any[]) => {
+  private proxyMethod(methodName: any) {
```

Comment:
Why are these changes necessary? Something to do with the latest TS version?

Reply:
Ye, the latest TS version was whining about this piece of code and it did not work the way it worked before (compiler errors). That's why I changed it. If you know a better solution please do implement it :) I am also not fond of the `any` keyword here.

Reply:
I'm curious as to what feature resulted in this extra check. Do you have an idea?

Reply:
I don't know to be honest, I got this error message:
![screenshot_20180626_075228](https://user-images.githubusercontent.com/1557855/41891744-edfb4f80-7915-11e8-8cca-020109f2bf17.png)